Doc/library/math.rst

```diff
@@ -298,9 +298,12 @@ Number-theoretic and representation functions
 
 .. function:: trunc(x)
 
-   Return the :class:`~numbers.Real` value *x* truncated to an
-   :class:`~numbers.Integral` (usually an integer). Delegates to
-   :meth:`x.__trunc__() <object.__trunc__>`.
+   Given a :class:`~numbers.Real` value *x*, return *x* with the fractional part
```

Comment:
The following is what I wrote on the issue.
```suggestion
   Return *x* with the fractional part
```

Reply:
Agree, this is consistent with floor() and ceil() and also more accurate because this function doesn't directly interact with the `numbers` module.